src/components/organisms/PageHeader/ClusterSelectionTable.tsx

```diff
@@ -37,10 +38,10 @@ export const ClusterSelectionTable: FC<CLusterSelectionTableProps> = ({setIsClus
   useEffect(() => {
     const clusterTableRows: ClusterTableRow[] = kubeConfigContexts.map(context => {
       const contextNamespaces = getNamespaces().filter(appNs => appNs.clusterName === context.name);
-      let hasFullAccess: boolean | undefined;
-      if (context.name === kubeConfigContext) {
-        hasFullAccess = clusterAccess?.every(ca => ca.hasFullAccess);
-      }
+      const clusterSpecificAccess = clusterAccess?.filter(ca => ca.context === context.name) || [];
+      const hasFullAccess = clusterSpecificAccess.length
+        ? clusterSpecificAccess?.every(ca => ca.hasFullAccess)
+        : undefined;
```

Comment:
```suggestion
      const hasFullAccess = clusterSpecificAccess?.every(ca => ca.hasFullAccess);
```
seems redundant to do more than this

Reply:
oh ok, didn't know that